engine/gamesys/src/gamesys/scripts/box2d/script_box2d_body.cpp

```diff
@@ -582,14 +572,11 @@ namespace dmGameSystem
         {"apply_linear_impulse", Body_ApplyLinearImpulse},
         {"apply_angular_impulse", Body_ApplyAngularImpulse},
 
-        {"get_next", Body_GetNext},
```

Comment:
This function does not exist, and there's not really a counterpart in box2d. We would have to implement this ourselves